firmware/common2015/utils/rj-macros.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+
+/* Macros for string conversion pass through 2 sets to help out
+ * with variable expansion. Not really sure if this is needed, but
+ * doing it anyways since there's pratically magic happening between
+ * the pre-processor and the log function in 'logger.hpp'.
+ */
+#define STRINGIFY(x)        #x
```

Comment:
One add commas, one doesn't. The STRINGIFY/TO_STRING thing has to be there for the logging stuff because there's a lot of prepreocessor magic happening for that.